docs/src/guides/micronaut/deploy/customize.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Customize Micronaut for Platform.sh"
+sidebarTitle: "Customize"
+weight: -90
+toc: false
+description: |
+    Add some helpful dependencies, and modify your Micronaut site to read from a Platform.sh environment.
+---
+
+Now that your code contains all of the configuration to deploy on Platform.sh, it's time to make your Micronaut site itself ready to run on a Platform.sh environment. There are a number of additional steps that are either required or recommended, depending on how well you want to optimize your site.
+
+## Install the Config Reader
+
+{{< guides/config-reader-info lang="java" >}}
+
+Below is an example of how to install the Config Reader for Java using Maven:
+
+```xml
+<dependency>
+    <groupId>sh.platform</groupId>
+    <artifactId>config</artifactId>
+    <version>2.2.2</version>
+</dependency>
+```
+
+and Gradle:
+
+```txt
+compile group: 'sh.platform', name: 'config', version: '2.2.2'
+```
+
+## `.environment`
+
+The `.platform.app.yaml` file on the [previous page](/guides/micronaut/deploy/configure.md#application-container-platformappyaml) has been pulled directly from the [Micronaut template](https://github.com/platformsh-templates/micronaut/blob/master/.platform.app.yaml). It is sufficient to deploy Micronaut on it's own, once Micronaut makes it possible to overwrite configurations without impacting the application itself, you might elect to rely more heavily on environment variables in it's place. 
```

Comment:
```suggestion
The `.platform.app.yaml` file on the [previous page](/guides/micronaut/deploy/configure.md#application-container-platformappyaml) has been pulled directly from the [Micronaut template](https://github.com/platformsh-templates/micronaut/blob/master/.platform.app.yaml). It is sufficient to deploy Micronaut on it's own, but since Micronaut makes it possible to overwrite configurations without impacting the application itself, you might elect to rely more heavily on environment variables in it's place. 
```